src/index.tsx

```diff
@@ -1,14 +1,26 @@
-import React from 'react';
+import { StrictMode } from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
It is a good practice to move this `globalStyles` declaration to a separate file, for example under `/src/theme/globalStyles.ts`